### ISSUE TITLE
make plugins available on AUR

### ISSUE THREAD

**Issue by cyrinux** [marvinkreis/rofi-plugins#13]
Hi, thanks for you cool tools.

Could you please publish them, using for example the good https://github.com/eli-schwartz/aurpublish

Thanks !

**Comment by marvinkreis** [comment]
Yes. I will split the repository in the near future, after that I will look into publishing the plugins to the AUR.

**Comment by cyrinux** [comment]
@marvinkreis Ok thanks :)

**Comment by marvinkreis** [comment]
Done :)
